library/classes/html2text.class.php

```diff
@@ -67,29 +67,29 @@
  *  for the _build_link_list() function: "https".
  *  Updated 04/06/05.
  *
- *  Thanks to Marc Bertrand (http://www.dresdensky.com/) for
+ *  Thanks to Marc Bertrand (https://www.dresdensky.com/) for
  *  suggesting a revision to the word wrapping functionality; if you
  *  specify a $width of 0 or less, word wrapping will be ignored.
  *  Updated 11/02/06.
  *
  *  *** Big housecleaning updates below:
  *
- *  Thanks to Colin Brown (http://www.sparkdriver.co.uk/) for
+ *  Thanks to Colin Brown (https://www.sparkdriver.co.uk/) for
  *  suggesting the fix to handle </li> and blank lines (whitespace).
- *  Christian Basedau (http://www.movetheweb.de/) also suggested the
+ *  Christian Basedau (https://www.movetheweb.de/) also suggested the
  *  blank lines fix.
  *
- *  Special thanks to Marcus Bointon (http://www.synchromedia.co.uk/),
- *  Christian Basedau, Norbert Laposa (http://ln5.co.uk/),
+ *  Special thanks to Marcus Bointon (https://www.synchromedia.co.uk/),
+ *  Christian Basedau, Norbert Laposa (https://ln5.co.uk/),
```

Comment:
This link (https://ln5.co.uk/) only works with http , so needs to stay as http

Reply:
👍